ibis/backends/flink/tests/conftest.py

```diff
@@ -30,21 +42,43 @@ def connect(*, tmpdir, worker_id, **kw: Any):
         string_class = gateway.jvm.String
         string_array = gateway.new_array(string_class, 0)
         env_settings = (
-            gateway.jvm.org.apache.flink.table.api.EnvironmentSettings.inBatchMode()
+            gateway.jvm.org.apache.flink.table.api.EnvironmentSettings.inStreamingMode()
+            if streaming_mode
+            else gateway.jvm.org.apache.flink.table.api.EnvironmentSettings.inBatchMode()
         )
         stream_env = gateway.jvm.org.apache.flink.streaming.api.environment.StreamExecutionEnvironment
         flink_cluster_addr = os.environ.get("FLINK_REMOTE_CLUSTER_ADDR", "localhost")
         flink_cluster_port = int(os.environ.get("FLINK_REMOTE_CLUSTER_PORT", "8081"))
-        j_stream_execution_environment = stream_env.createRemoteEnvironment(
+        j_execution_environment = stream_env.createRemoteEnvironment(
             flink_cluster_addr,
             flink_cluster_port,
             env_settings.getConfiguration(),
             string_array,
         )
 
-        env = StreamExecutionEnvironment(j_stream_execution_environment)
-        stream_table_env = StreamTableEnvironment.create(env)
-        return ibis.flink.connect(stream_table_env, **kw)
+        env = StreamExecutionEnvironment(j_execution_environment)
+        table_env = StreamTableEnvironment.create(env)
+        table_config = table_env.get_config()
+        table_config.set("table.local-time-zone", "UTC")
+
+        return table_env
+
+
+class TestConf(BackendTest):
+    force_sort = True
+    deps = "pandas", "pyflink"
+
+    @staticmethod
+    def connect(*, tmpdir, worker_id, **kw: Any):
+        """Flink backend is created in batch mode by default. This is to
+        comply with the assumption that the tests under ibis/ibis/backends/tests/
+        are for batch (storage or processing) backends.
+        """
+        import os
+
+        local_env = os.getenv("IBIS_FLINK_LOCAL_ENV", "False").lower() in ("true", "1")
```

Comment:
When would we ever set this to `false`?

Reply:
I guess you are referring to the fact that `IBIS_FLINK_LOCAL_ENV` is binary, and we can simply interpret it being defined as the flag to run the tests locally. I chose to make it explicit with `true` and `false`, mostly due to personal preference over possibly having to `unset` env variables.

Reply:
Sounds good, let's just delete the variable then :)